modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -382,21 +382,33 @@ With this update, when auto focus is set, the editor is focused and scrolled int
 
 === Fixed `contenteditable="false"` lists and `contenteditable="true"` list elements issues
 
-`contenteditable="false"` lists can no longer be toggled; and contenteditable="true" list elements within these lists can no longer be indented, split into another list element, or appended to the previous list element by deletion.
+Previously, lists with `contenteditable="false"` attributes could be toggled and updated. And, within those lists, list elements with `contenteditable="true"` attributes could be indented, split into another list element, or appended to the previous list element by deletion.
+
+Neither behavior was correct and, as of {productname} 6.2, this has been corrected.
+
+In {productname} 6.2, list functionality in this regard is dependent on the value of the list’s `contenteditable` attribute.
+
+And an editable list element’s functionality is now dependent on the value of the `contenteditable` attribute in its parent list.
```

Comment:
More specifically, this is dependent on the contenteditable state, meaning that the parent list may not have a contenteditable attribute (no attribute specified, by default it would have `inherit`) but the first parent of that list with a contenteditable attribute determines the contenteditable state of that list, for example, a noneditable `div` means any list within that div without a contenteditable attribute would be noneditable. Another example would be a contenteditable true list would be editable, even within a noneditable div. Just wanted to clarify that this isn't always determined by the attribute, but by the state of the first parent with the attribute.

Reply:
Re-wrote this section to make the above explicit.